SDKBuildScripts/Update-PlayFabSdk.ps1

```diff
@@ -0,0 +1,160 @@
+[CmdletBinding(SupportsShouldProcess = $true, DefaultParameterSetName = "ApiSpecGitUrl")]
+param(
+    [Parameter(Position = 1)]
+    [ValidateSet(
+        "ActionScriptSDK",
+        "Cocos2d-xSDK",
+        "CSharpSDK",
+        "JavaSDK",
+        "JavaScriptSDK",
+        "LuaSDK",
+        "NodeSDK",
+        "Objective_C_SDK",
+        "PhpSDK",
+        "PostmanCollection",
+        "PythonSDK",
+        "SdkTestingCloudScript",
+        "UnrealMarketplacePlugin",
+        "UnitySDK",
+        "WindowsSDK",
+        "XPlatCppSDK")]
+    [string[]]$SdkNames,
+    [Parameter(ParameterSetName="ApiSpecPath")]
+    [string]$ApiSpecPath,
+    [Parameter(ParameterSetName="ApiSpecPfUrl")]
+    [string]$ApiSpecPfUrl,
+    [Parameter(ParameterSetName="ApiSpecPfUrlBranch")]
+    [string]$ApiSpecBranch,
+    [Parameter(ParameterSetName="ApiSpecGitUrl")]
+    [switch]$ApiSpecGitUrl,
+    [switch]$KeepSource,
+    [switch]$NonNullable,
+    [switch]$Beta
+)
+
+begin
+{
+    if(!(Get-Command node))
+    {
+        throw "You must have Node.js installed to generate the SDK"
+    }
+
+    $sdkTargetSrcMap = @{
+        "ActionScriptSDK" = "actionscript";
+        "Cocos2d-xSDK" = "cpp-cocos2dx";
+        "CSharpSDK" = "csharp";
+        "JavaSDK" = "java";
+        "JavaScriptSDK" = "javascript";
+        "LuaSDK" = "LuaSdk";
+        "NodeSDK" = "js-node";
+        "Objective_C_SDK" = "objc";
+        "PhpSDK" = "PhpSdk";
+        "PostmanCollection" = "postman";
+        "PythonSDK" = "PythonSdk";
+        "SdkTestingCloudScript" = "SdkTestingCloudScript";
+        "UnrealMarketplacePlugin" = "UnrealMarketplacePlugin";
+        "UnitySDK" = "unity-v2";
+        "WindowsSDK" = "windowssdk";
+        "XPlatCppSDK" = "xplatcppsdk";
+    }
+
+    $sdksPath = Resolve-Path (Join-Path $PSScriptRoot "..\..\sdks")
+    if(!(Test-Path $sdksPath))
+    {
+        mkdir $sdksPath
+    }
+
+    if($ApiSpecBranch)
+    {
+        $ApiSpecPfUrl = "https://$($ApiSpecBranch).playfabapi.com/apispec"
```

Comment:
This URL is incomplete and mis-labeled.
It should properly be:
https://($verticalName).$(clusterName).playfabapi.com/apispec"
There's also alternate formats that apply when one or the other is undefined.

I'd go so far as to say this is at minimum, misleading.
(I also understand that you're probably reading from other public code, which is also misleading in the same way, and we're trying to fix that).

Reply:
@pgilmorepf I'm fine with changing this.  This was literally intended to be almost identical to the existing scripts, but I'm renaming stuff to make it much more clear so I'm happy to make improvements.

However I think you got one of the cloud/vertical things backwards.  If you specify https://foo.playfabapi.com you're looking at the foo vertical in the main cloud.  You can verify this by looking at `https://spi-economy.spi.playfabapi.com/healthstatus` vs `https://spi-economy.playfabapi.com/healthstatus`.  In both of these cases, it returns the vertical `spi-economy`.